lib/typed_env.rb

```diff
@@ -0,0 +1,9 @@
+class TypedEnv
+  class << self
+    TRUTHY_VALUES = %w(t true yes y 1).freeze
```

Comment:
Just noting that our docs should recommend setting to `true` or `false` only.

Reply:
Thanks, I made a reference to this in the docs ticket: https://github.com/cyberark/docs-cyberark-conjur-service-broker/issues/4